packages/slice-machine/components/LoginModal/index.tsx

```diff
@@ -25,15 +25,16 @@ import { ModalKeysEnum } from "@src/modules/modal/types";
 import { getEnvironment } from "@src/modules/environment";
 import useSliceMachineActions from "@src/modules/useSliceMachineActions";
 import Tracker from "@src/tracker";
-
-Modal.setAppElement("#__next");
+import { preferWroomBase } from "../../server/src/api/common/utils";
```

Comment:
I think as this fonction is shared between the front and the backend it should be move to lib folder.

Reply:
it's been moved

Reply:
i think I moved it to the src folder :/